### ISSUE TITLE
Pytorch model doesn't produce the intended output

### ISSUE THREAD

**Issue by Samir55** [junyanz/interactive-deep-colorization#63]
Hello authors,
I am encountering a problem that I couldn’t reproduce the same output by
training your model on the ILSVRC2012 training set. The model losses
(G_CE And G_entr) tend to stabilize around 3.1 across epochs. Also,
the colorized images tend to have a somehow yellowish color. I trained
the model for 6 epochs. I am using Python 3.5 and CUDA 9.1 with GTX
1060 GPU. Can you help me discover what’s going on? I am adding
here some output pictures and the loss graph. Thanks for your time and
efforts.

![00000195_0p031_fake_reg](https://user-images.githubusercontent.com/18165914/50447704-bcf31d00-0925-11e9-93e7-17f51a7bc0b1.png)
![00000198_0p125_fake_reg](https://user-images.githubusercontent.com/18165914/50447705-bfee0d80-0925-11e9-9043-53a32686eedf.png)
![loss_graph](https://user-images.githubusercontent.com/18165914/50447714-d85e2800-0925-11e9-90d9-ebdfa84aacfc.png)

**Comment by richzhang** [comment]
I believe I did not do the initialization properly in the pytorch code. I am looking into it. Thanks for your patience.
